src/components/Input/index.js

```diff
@@ -144,11 +147,19 @@ class Input extends Component {
         ref={ref => {
           this.inputContainer = ref
         }}
+        style={{
```

Comment:
According to the codebase, it is better to use as much as possible `className` to style instead of inline.